### PR TITLE
update grafana-operator to v3.8.1 and amo to 1.5.0

### PR DESCRIPTION
**Pull request by HubertStefanski**
- Update grafana opeator to v3.8.1
- Bump amo to 1.5.0
- Add note to README regarding maintenance

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,9 +13,9 @@ COMPILE_TARGET=./tmp/_output/bin/$(PROJECT)
 # You can delete this comment afterwards.
 PROMETHEUS_OPERATOR_VERSION=v0.34.0
 LOCAL=local
-GRAFANA_OPERATOR_VERSION=v3.6.0
-AMO_VERSION=1.4.0
-PREV_AMO_VERSION=1.3.0
+GRAFANA_OPERATOR_VERSION=v3.8.0
```
[the remark this answers is not in the document]
Comment (HubertStefanski) [reply Makefile]: It should, good catch, Vscode didn't save the change in that file for whatever reason, :thumbsup

**Comment by CathalOConnorRH** [comment]
Probably need to update PR title version number and description to include 3.8.1 but otherwise looks good
/lgtm

**Comment by HubertStefanski** [comment]
> Probably need to update PR title version number and description to include 3.8.1 but otherwise looks good
> /lgtm

@CathalOConnorRH Done :+1:

File: deploy/olm-catalog/application-monitoring-operator/1.5.0/application-monitoring-operator.v1.5.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+  - Monitoring
+  - AlertManager
+  maintainers:
+  - email: placeholder@email.com
+    name: Integreatly
+  maturity: alpha
+  provider:
+    name: Red Hat
+  replaces: application-monitoring-operator.v1.4.0
+  version: test
```
Comment (grdryn) [inline deploy/olm-catalog/application-monitoring-operator/1.5.0/application-monitoring-operator.v1.5.0.clusterserviceversion.yaml:336]: Should this be `version: 1.5.0`?

Comment (HubertStefanski) [reply deploy/olm-catalog/application-monitoring-operator/1.5.0/application-monitoring-operator.v1.5.0.clusterserviceversion.yaml]: fixed

**Review by grdryn — APPROVED** [review]
/lgtm
/approve

**Pull request merged**